gulpfile.js

```diff
@@ -427,108 +181,38 @@ gulp.task('add-host-opera', function() {
 	return addHostToManifest('Opera/includes/loader.js');
 });
 
-gulp.task('module-css-opera', function() {
-	return gulp.src('lib/modules/*.css')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'modules')));
-});
-
-gulp.task('core-css-opera', function() {
-	return gulp.src('lib/core/*.css')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'core')));
-});
-
-gulp.task('vendor-css-opera', function() {
-	return gulp.src('lib/vendor/*.css')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'vendor')));
-});
-
-gulp.task('lib-js-opera', function() {
-	return gulp.src('lib/*.js')
-		.pipe(gulp.dest(path.join(operaBuildDir)));
-});
-
-gulp.task('vendor-js-opera', function() {
-	return gulp.src('lib/vendor/*.js')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'vendor')));
-});
-
-gulp.task('core-js-opera', function() {
-	return gulp.src('lib/core/*.js')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'core')));
-});
-
-gulp.task('modules-js-opera', function() {
-	return gulp.src([ 'lib/modules/**/*.js' , 'lib/modules/**/*.json' ])
-		.pipe(gulp.dest(path.join(operaBuildDir, 'modules')));
-});
-
-gulp.task('opera-js-opera', function() {
-	return gulp.src('Opera/*.js')
-		.pipe(gulp.dest(path.join(operaBuildDir)));
-});
-
-gulp.task('includes-js-opera', function() {
-	return gulp.src('Opera/includes/*.js')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'includes')));
-});
-
-gulp.task('root-images-opera', function() {
-	return gulp.src('OperaBlink/*.gif')
-		.pipe(gulp.dest(path.join(operaBuildDir)));
-});
-
-gulp.task('opera-move-1', function() { // move other stuff that doesn't fit elsewhere
-	return gulp.src('lib/core/*.html')
-		.pipe(gulp.dest(path.join(operaBuildDir, 'core')));
-});
-
-gulp.task('opera-move-2', function() {
-	return gulp.src('Opera/*.html')
-		.pipe(gulp.dest(path.join(operaBuildDir)));
-});
-
-gulp.task('opera-move-3', function() {
-	return gulp.src('Opera/*.xml')
-		.pipe(gulp.dest(path.join(operaBuildDir)));
-});
-
-gulp.task('opera-move-4', function() {
-	return gulp.src('package.json')
-		.pipe(gulp.dest(path.join(operaBuildDir)));
-});
-
 // Zip tasks
 gulp.task('chrome-zip', function() {
-	return gulp.src(path.join(chromeBuildDir, '**/*'))
+	return gulp.src(path.join(buildDir.chrome, '**/*'))
 		.pipe(zip('chrome.zip'))
 		.pipe(gulp.dest(zipDir));
 });
 
 gulp.task('safari-zip', function() {
```

Comment:
Is there a way to combine zip tasks into a single one that takes in a browser parameter?